types/operator.go

```diff
@@ -88,9 +89,50 @@ type StakeAmount = *big.Int
 
 // OperatorId is the ID of an operator, defined by the AVS registry
 // It is currently the hash of the operator's G1 pubkey (in the bls pubkey registry)
-type OperatorId = [32]byte
-type QuorumNum = uint8
-type QuorumThresholdPercentage = uint32
+type OperatorId = Bytes32
+type QuorumNums []QuorumNum
+
+func (q QuorumNums) LogValue() slog.Value {
+	return slog.StringValue(fmt.Sprintf("%v", q))
+}
+
+func (q QuorumNums) Underlying() []uint8 {
```

Comment:
can there be a better name? when I read `Underlying()` in code, I am like what does it even mean? like the above `GetUnderlyingBytes` is self explanatory.

Reply:
It's not bytes though, it returns the underlying type.
The idea is that eigensdk wraps normal types like `type QuorumNum int8` so that we can define methods on QuorumNum. But the contract bindings expect the unwrapped (underlying) type, hence the name `underlying()`. I think it makes sense?
Could also potentially call it `Unwrap()`?

Reply:
ye `Unwrap` maybe is better. but i just feel `underlying` doesn't say anything. `underlyingType`?

Reply:
[d4cc682](https://github.com/Layr-Labs/eigensdk-go/pull/146/commits/d4cc68267f1dd2d04485c35350bfeaf6b72507d6)